stratum/hal/lib/barefoot/barefoot.bzl

```diff
@@ -67,8 +67,20 @@ pkg_tar(
   name = "bf_shareable_files",
   srcs = glob(["barefoot-bin/share/microp_fw/**"]) +
          glob(["barefoot-bin/share/bfsys/**"]) +
-         glob(["barefoot-bin/share/tofino_sds_fw/**"]),
+         glob(["barefoot-bin/share/tofino_sds_fw/credo/**"]) +
+         [
+            "barefoot-bin/share/tofino_sds_fw/avago/firmware/pcie_serdes.0x1066_2147.rom",
+            "barefoot-bin/share/tofino_sds_fw/avago/firmware/sbus_master.0x101D_2001.rom",
+            "barefoot-bin/share/tofino_sds_fw/avago/firmware/serdes.0x106F_204D.rom",
+            "barefoot-bin/share/tofino_sds_fw/avago/firmware/serdes.0x1091_244D.rom",
+         ],
   mode = "0644",
+  symlinks = {
```

Comment:
Can these be dynamically generated in a macro? I don't remember if globs are expanded in macro code or not.